### DISCUSSION TITLE
Can I skip calculating metrics with incomplete information without throwing an error?

### DISCUSSION THREAD

**Discussion by gotoariel** [rasbt/mlxtend#1008] [Q&A]
I have a relatively small dataset with large itemsets up to size 70. I'm trying to use fpmax because otherwise there's way too many subsets of itemsets overlapping. However as soon as I switch from apriori to fpmax, I get KeyErrors because (I think) of incomplete support information.

I'm wondering if there's a way I can just skip the offending itemsets instead of it stopping the whole process. I'm not sure how useful `support_only` will be if I can't sort by confidence or conviction at the end. Actually, what does `association_rules` with `support_only` do that's different from the support already generated in the `frequent_itemset` dataframe?

**Comment by rasbt** [comment]
Yeah, I agree that having an option to skip the offending itemsets would be ideal, but this is not implemented. I think the best workaround at the moment is using `support_only=True` and then computing the other metrics manually afterwards
